components/ui/chat-settings-form.tsx

```diff
@@ -100,12 +100,22 @@ const AdvancedContent: FC<AdvancedContentProps> = ({
   onChangeChatSettings,
   showTooltip
 }) => {
-  const { profile, selectedWorkspace } = useContext(ChatbotUIContext)
+  const {
+    profile,
+    selectedWorkspace,
+    availableOpenRouterModels,
+    selectedAssistant
+  } = useContext(ChatbotUIContext)
+
+  function findOpenRouterModel(modelId: string): any {
```

Comment:
I noticed that you are using the type 'any.' Is it possible for you to implement specific types for it?

interface OpenRouterModel {
  modelId: string;
}

function findOpenRouterModel(modelId: string): OpenRouterModel | undefined

Reply:
Done